common/models/product.json

```diff
@@ -33,10 +36,16 @@
       "description": "the listing wineries"
     },
     "grapes": {
-      "type": "hasAndBelongsToMany",
+      "type": "referencesMany",
```

Comment:
using our old way to solve this. so the sample data creation works as implemented.